### DISCUSSION TITLE
Disable compress source when compiling extension

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#14527] [Archive]
I have the problem, that my extensions I am developing throw errors. The problem is that is always very hard to find the issue, because directus compress the source. Is it possible to deactivate / workaround this?

**Comment by br41nslug** [comment]
You can disable minification and enable sourcemaps using commandline parameters when building your extension.
- `--no-minify` disable minification
- `--sourcemap` include source maps in output

So `npm run build --no-minify --sourcemap` should help

**Reply by rokdd** [reply]
I will try but I assume it will work ;-) Thank you so much
